README.md

```diff
@@ -1,18 +1,1481 @@
+Starts a new unlock recovery transaction for a given user and issues a recovery token that can be used to unlock a user’s account.
+
+  - `username` - User’s non-qualified short-name (e.g. dade.murphy) or unique fully-qualified login (e.g dade.murphy@example.com)
+  - `factorType` - Recovery factor to use for primary authentication. May be omitted if a trusted application is whitelisted.
+  - `relayState` - Optional state value that is persisted for the lifetime of the recovery transaction
+
+```javascript
+authClient.unlockAccount({
+  username: 'dade.murphy@example.com',
+  factorType: 'SMS', // 'EMAIL' or undefined for trusted applications
+})
+.then(function(transaction) {
+  // if factorType is undefined
+  // return transaction.recovery({
+  //   recoveryToken: '00xdqXOE5qDZX8-PBR1bYv8AESqIFinDy3yul01tyh'
+  // });
+
+  return transaction.verify({
+    passCode: '123456' // The passCode from the SMS
+  });
+})
+.then(function(transaction) {
+  if (transaction.status === 'SUCCESS') {
+    authClient.session.setCookieAndRedirect(transaction.sessionToken);
+  } else {
+    throw 'We cannot handle the ' + transaction.status + ' status';
+  }
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## verifyRecoveryToken(options)
+
+Validates a recovery token that was distributed to the end-user to continue the recovery transaction.
+
+- `recoveryToken` - Recovery token that was distributed to end-user via out-of-band mechanism such as email
+
+```javascript
+authClient.verifyRecoveryToken({
+  recoveryToken: '00xdqXOE5qDZX8-PBR1bYv8AESqIFinDy3yul01tyh'
+})
+.then(function(transaction) {
+  if (transaction.status === 'SUCCESS') {
+    authClient.session.setCookieAndRedirect(transaction.sessionToken);
+  } else {
+    throw 'We cannot handle the ' + transaction.status + ' status';
+  }
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## tx.resume()
+
+Resumes an in-progress transaction. This is useful if a user navigates away from the login page before authentication is complete.
+
+```javascript
+authClient.tx.exists()
+.then(function(exists) {
+  if (exists) {
+    return authClient.tx.resume();
+  }
+  throw new Error('a session does not exist');
+})
+.then(function(transaction) {
+  console.log('current status:', transaction.status);
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## tx.exists()
+
+Check for a transaction to be resumed. This is synchronous and returns `true` or `false`.
+
+```javascript
+authClient.tx.exists()
+.then(function(exists) {
+  if (exists) {
+    console.log('a session exists');
+  } else {
+    console.log('a session does not exist');
+  }
+})
+.fail(function(err) {
+  console.error(err);
+});
+```
+
+## transaction.status
+
+When Auth Client methods resolve, they return an object that encapsulates the new state in the authentication flow, a **transaction** object. This object contains metadata about the current state, and methods that can be used to progress to the next state.
```

Comment:
Also, it seems like this would be a good point to link to:
http://developer.okta.com/docs/api/resources/authn.html#transaction-state

Or maybe even include the transaction model diagram.